### PR TITLE
Removed Treasure Island geometry from FiDi

### LINKED ISSUE #126: Does the Financial District (SF) really include Treasure Island
I mean maybe it's like "The City" in London, but probably not...

https://whosonfirst.mapzen.com/spelunker/id/85865899/

screen shot 2015-10-26 at 08 06 46

### PR DESCRIPTION
**Pull request by stepps00**
Fixes #126. Treasure Island geometry was removed from the FiDi record. Requesting merge to Master.

### CODE REVIEW

**Comment by nvkelso** [comment]
The new geometry looks good, but...

```
"src:geom":"zetashapes",
"src:geom_alt":[
    "quattroshapes"
],
```

Is now inconsistent. This is a good one for Boundary Issues to think about when supporting geometry uploads, so /cc @dphiffer and @thisisaaronland.

* We should specify `src:geom` as **mapzen** since it's our custom shape now.
* We should store the original zetashape (pre your modifications) into a new alt geom file, and
* We should add **zetashapes** to the `src:geom_alt` list.

**Comment by stepps00** [comment]
Exportified existing geojson file and updated the `src` fields to reflect **mapzen** as the `src:geom` value and **zetashapes** as the `src:geom_alt` value.

Created new zetashapes alt_geometry .geojson and exportified.

File: data/858/658/99/85865899.geojson [hunk 1]
```diff
@@ -50,9 +50,9 @@
     "name:eng_x_variant":[
         "Financialdistrict"
     ],
-    "src:geom":"zetashapes",
+    "src:geom":"mapzen",
     "src:geom_alt":[
-        "quattroshapes"
+        "zetashapes"
```
Comment (nvkelso) [inline data/858/658/99/85865899.geojson]: it should be

```
    "quattroshapes",
    "zetashapes"
```

Comment (stepps00) [inline data/858/658/99/85865899.geojson]: Added in `"quattroshapes"` as a `geom_alt` in latest commit.

**Comment by nvkelso** [comment]
LGTM

**Comment by thisisaaronland** [comment]
👍

**Pull request merged**